integration/network/ipvlan/ipvlan_test.go

```diff
@@ -569,8 +569,7 @@ func TestIPVlanDNS(t *testing.T) {
 
 	net.StartDaftDNS(t, "127.0.0.1")
 
-	tmpFileName := net.WriteTempResolvConf(t, "127.0.0.1")
-	d := daemon.New(t, daemon.WithEnvVars("DOCKER_TEST_RESOLV_CONF_PATH="+tmpFileName))
+	d := daemon.New(t, daemon.WithResolvConf(net.GenResolvConf(t, "127.0.0.1")))
```

Comment:
Heh, very unrelated, but we should probably avoid using `net` as alias, as it would be clashing with Go's `net` package;

```go
net "github.com/docker/docker/integration/internal/network"
```